src/qt/addresstablemodel.cpp

```diff
@@ -8,7 +8,7 @@
 #include <qt/guiutil.h>
 #include <qt/walletmodel.h>
 
-#include <interfaces/node.h>
+#include <interfaces/wallet.h>
```

Comment:
why?

Reply:
Not sure where this came from but does not longer exist this way